engine/src/main/java/io/zeebe/engine/state/appliers/DeploymentDistributing.java

```diff
@@ -0,0 +1,30 @@
+/*
+ * Copyright Camunda Services GmbH and/or licensed to Camunda Services GmbH under
+ * one or more contributor license agreements. See the NOTICE file distributed
+ * with this work for additional information regarding copyright ownership.
+ * Licensed under the Zeebe Community License 1.0. You may not use this file
+ * except in compliance with the Zeebe Community License 1.0.
+ */
+package io.zeebe.engine.state.appliers;
+
+import io.zeebe.engine.state.TypedEventApplier;
+import io.zeebe.engine.state.ZeebeState;
+import io.zeebe.engine.state.mutable.MutableDeploymentState;
+import io.zeebe.protocol.impl.record.value.deployment.DeploymentDistributionRecord;
+import io.zeebe.protocol.record.intent.DeploymentDistributionIntent;
+
+public class DeploymentDistributing
+    implements TypedEventApplier<DeploymentDistributionIntent, DeploymentDistributionRecord> {
```

Comment:
We should align the name with the record name `Distribution`. And add the suffix `EventApplier` :wink: 

```suggestion
public class DeploymentDistributionEventApplier
    implements TypedEventApplier<DeploymentDistributionIntent, DeploymentDistributionRecord> {
```

Reply:
Hm the intent is `Distributing` and the record is `DeploymentDistribution`. That is the reason why I called it `DeploymentDistributing` any thoughts on this @saig0

Reply:
Following the schema (record + intent + "EventApplier"), we should name it `DeploymentDistributionDistributingEventApplier` :joy: 

I'm fine with `DeploymentDistributingEventApplier`.

Reply:
Fine with DeploymentDistributingApplier as well ? The others are also only called *Applier :)

Reply:
Good point. Let's call it `DeploymentDistributingApplier`.

Reply:
Thanks :+1: